src/main/java/mate/academy/lib/ConnectionUtil.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.lib;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/test";
```

Comment:
The `DB_URL` is incomplete. It should specify the database server's URL, port, and the database name you're connecting to. For example, `jdbc:mysql://localhost:3306/myDatabase` where `myDatabase` is the name of your database.